src/__tests__/FileUpload.test.tsx

```
@@ -1,851 +1,827 @@
-      render(<FileUpload onFileSelect={mockOnFileSelect} />);
+      // Force a rerender with invalid props
+      expect(() => {
+        rerender(<FileUpload onFileSelect={null} />);
+      }).not.toThrow();
+    });
+  });
+
+  describe('Browser Compatibility Edge Cases', () => {
+    it('works when File API is not fully supported', async () => {
+      const originalFile = global.File;
+      delete global.File;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)